docs/unretire-anyone.rst

```diff
@@ -22,3 +22,10 @@ Usage
 ::
 
     unretire-anyone
+
+Options:
```

Comment:
Check out https://docs.dfhack.org/en/stable/docs/Documentation.html#options for how to format the options section